en.lproj/SGLocalizable.strings

```diff
@@ -3,22 +3,22 @@
 "Settings.Tabs.Header" = "TABS";
 "Settings.Tabs.HideTabBar" = "Hide Tab bar";
 "Settings.Tabs.ShowContacts" = "Show Contacts Tab";
-"Settings.Tabs.ShowNames" = "Show tab names";
+"Settings.Tabs.ShowNames" = "Show Tab Names";
 
-"Settings.Folders.BottomTab" = "Folders at bottom";
-"Settings.Folders.BottomTabStyle" = "Bottom folders style";
+"Settings.Folders.BottomTab" = "Folders at Bottom";
+"Settings.Folders.BottomTabStyle" = "Bottom Folders Style";
 
 /* Do not translate */
 "Settings.Folders.BottomTabStyle.ios" = "iOS";
 /* Do not translate */
 "Settings.Folders.BottomTabStyle.telegram" = "Telegram";
 
-"Settings.Folders.RememberLast" = "Open last folder";
-"Settings.Folders.RememberLast.Notice" = "Swiftgram will open the last used folder after restart or account switch";
+"Settings.Folders.RememberLast" = "Open Last Folder";
+"Settings.Folders.RememberLast.Notice" = "Open the last folder when you switch accounts or restart Swiftgram.";
```

Comment:
The accent here is that Swiftgram will do that automatically. I think we're missing it now.